### ISSUE TITLE
fix compile time warnings

### ISSUE THREAD

**Issue by naturallymitchell** [patchworkuniversity/torchbear#23]
I think unused bindings could be turned off with #feature cfg.

**Comment by Arnaz87** [comment]
Yeah, this is a good idea. logging broke when I tried this though, but nonetheless it's a good idea.
